source/jsonnet/england-wales/individual/blocks/identity-and-health/national_identity.jsonnet

```diff
@@ -99,6 +99,23 @@ function(region_code) {
     },
   ],
   routing_rules: [
+    {
+      goto: {
+        block: 'national-identity-additional-other',
+        when: [
+          {
+            id: 'national-identity-answer',
+            condition: 'contains any',
+            values: ['British', 'English', 'Welsh', 'Scottish', 'Northern Irish'],
+          },
+          {
+            id: 'national-identity-answer',
+            condition: 'contains',
+            value: 'Other',
+          },
```

Comment:
Am I reading this right - doesn't every option route to the same place? Does it need the when rules?

Reply:
Ah, actually I see now. Don't mind me.